app/src/main/java/com/pr0gramm/app/ui/views/CommentsAdapter.java

```diff
@@ -182,8 +184,21 @@ public CommentView(View itemView) {
         }
 
         public void setCommentDepth(int depth) {
-            MarginLayoutParams params = (MarginLayoutParams) itemView.getLayoutParams();
-            params.leftMargin = baseLeftMargin * depth;
+            LinearLayout commentWrapper = (LinearLayout) itemView.findViewById(R.id.comment_wrapper);
+
+            // Add missing spacers
+            for (int i = commentWrapper.getChildCount(); i < depth; i++) {
+                FrameLayout spacer = new FrameLayout(itemView.getContext());
+                LinearLayout.LayoutParams layoutParams = new LinearLayout.LayoutParams(2, ViewGroup.LayoutParams.MATCH_PARENT);
+                layoutParams.leftMargin = baseLeftMargin;
+                spacer.setLayoutParams(layoutParams);
+                spacer.setBackgroundColor(itemView.getResources().getColor(R.color.tag_background));
+                commentWrapper.addView(spacer, 0);
```

Comment:
Der ganze Code in der Schleife ist zwingend nötig da jeder Spacer neu erstellt werden muss.